Duplicati/Server/webroot/ngax/scripts/services/AppUtils.js

```diff
@@ -302,6 +314,10 @@ backupApp.service('AppUtils', function($rootScope, $timeout, $cookies, DialogSer
             if (d['--' + k] !== undefined) {
                 DialogService.dialog(gettextCatalog.getString('Error'), gettextCatalog.getString('Duplicate option {{opt}}', { opt: k }));
                 return false;
+            } else if (k == 'include' || k == 'exclude') {
+                // Cannot specify filters in extra options
+                DialogService.dialog(gettextCatalog.getString('Error'), gettextCatalog.getString('Cannot specify filter include or excludes in extra options'));
+                return false;
             }
```

Comment:
When is this code necessary ? how can filters get into extra options ? and I don't see it called when test-filter is called anyway.